packages/editor/src/components/rich-text/index.js

```diff
@@ -1000,7 +999,7 @@ RichText.contextTypes = {
 };
 
 RichText.defaultProps = {
-	formattingControls: DEFAULT_FORMATS,
+	formattingControls: FORMATTING_CONTROLS.map( ( { format } ) => format ),
```

Comment:
@iseulde - as you pointed out, we don't need this constant as we can produce it from the default `FORMATTING_CONTROLS`.